src/main/java/creativei/entity/Locality.java

```diff
@@ -14,6 +16,13 @@
     @ManyToOne
     private City city;
 
+    public Locality(){}
+
+    public Locality(AddressVo addressVo){
```

Comment:
Why do you need the whole addressVo here? Pass multiple params like (locationId, area, pincode)

Reply:
okay Issue resolved in commit  4e98a52

Reply:
more accurately constructor in present in this commit f4529da